lib/msf/core/post/vcenter/vcenter.rb

```diff
@@ -348,160 +352,22 @@ def get_vecs_private_key(vecs_store, entry_alias)
-          return nil if vpxuser_rows.nil?
-
-          vpxuser_rows = vpxuser_rows.split("\n")
-          return nil unless vpxuser_rows.first
-
-          vpxuser_rows.each do |vpxuser_row|
-            row_data = vpxuser_row.split('|')
-            next if row_data.length < 4 # shoudld always be 4 based on query, but this will catch 'command not found' or other things like that
-
-            user = {
-              'fqdn' => row_data[0],
-              'ip' => row_data[1],
-              'user' => row_data[2]
-            }
-
-            vpxuser_secret_b64 = row_data[3].gsub('*', '')
-            user['password'] = vpx_aes_decrypt(vpxuser_secret_b64, vc_sym_key).gsub('\"', '"')
-            output.append(user)
-          end
-          output
-        end
-
-        #
-        # Returns a list of virtual machines located on the server
-        # @param pg_password [String] postgress password
-        # @param vcdb_user [String] virtual center database username
-        # @param vcdb_name [String] virtual center database name
-        # @param vc_sym_key [String] sym key from virtual center
-        # @return [Array] list of hash tables where each table is a user, nil on error
-        #
-        def get_vpx_vms(pg_password, vcdb_user, vcdb_name, _vc_sym_key)
-          return nil unless command_exists? psql_bin
-
-          output = []
-          vm_rows = cmd_exec("#{postgress_connect(pg_password, vcdb_user, vcdb_name)} -c 'SELECT vmid, name, configfilename, guest_state, is_template FROM vpxv_vms;' -P pager -A -t")
-          return nil if vm_rows.nil?
-
-          vm_rows = vm_rows.split("\n")
-          return nil unless vm_rows.first
-
-          vm_rows.each do |vm_row|
-            row_data = vm_row.split('|')
-            next if row_data.length < 5 # shoudld always be 5 based on query, but this will catch 'command not found' or other things like that
-
-            vm = {
-              'vmid' => row_data[0],
-              'name' => row_data[1],
-              'configfilename' => row_data[3],
-              'guest_state' => row_data[4],
-              'is_template' => row_data[5]
-            }
-            output.append(vm)
-          end
-          output
-        end
-
         #
         # Returns a hash table of the vcdb.properties file
         # @param location [String] where the file is located. defaults to /etc/vmware-vpx/vcdb.properties
         # @return [Hash] hash of the file contents, nil on error
         #
-        def process_vcdb_properties_file(location = '/etc/vmware-vpx/vcdb.properties')
+        def process_vcdb_properties_file(location = vcd_properties_file)
           return nil unless file_exist?(location)
 
           contents = read_file(location)
           return nil if contents.nil?
 
+          if location == vcd_properties_file && is_root? == false
```

Comment:
It looks like the tests are failing because `is_root?` is not defined in this context:
```
Msf::Post::Vcenter::Vcenter FF

  1) Msf::Post::Vcenter::Vcenter gets vcdb properties and processes them correctly
     Failure/Error: if location == vcd_properties_file && is_root? == false

     NoMethodError:
       undefined method `is_root?' for #<Msf::Module:0x00007fb1ec0ed558 @module_info_copy={}, @module_info={"Name"=>"No module name", "Description"=>"No module description", "Version"=>"0", "Author"=>nil, "Arch"=>nil, "Platform"=>[], "Ref"=>nil, "Privileged"=>false, "License"=>"Metasploit Framework License (BSD)", "Notes"=>{}, "Compat"=>{"Encoder"=>{}, "Meterpreter"=>{}, "Nop"=>{}, "Payload"=>{}}}, @uuid="tuo9jwfs", @module_store={}, @author=[], @arch=[], @platform=#<Msf::Module::PlatformList:0x00007fb1ec0dec60 @platforms=[]>, @references=[], @options={"WORKSPACE"=>#<Msf::OptString:0x00007fb1ec0c0d00 @name="WORKSPACE", @advanced=true, @evasion=false, @aliases=[], @max_length=nil, @conditions=[], @fallbacks=[], @required=false, @desc="Specify the workspace for this module", @default=nil, @enums=[], @owner=Msf::Module>, "VERBOSE"=>#<Msf::OptBool:0x00007fb1ec0bf798 @name="VERBOSE", @advanced=true, @evasion=false, @aliases=[], @max_length=nil, @conditions=[], @fallbacks=[], @required=false, @desc="Enable detailed status messages", @default=false, @enums=[], @owner=Msf::Module>}, @datastore={"WORKSPACE"=>nil, "VERBOSE"=>false}, @privileged=false, @license="Metasploit Framework License (BSD)">
```